src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -163,10 +163,32 @@ public void linkDefendant(String authorisation) {
             ).forEach(caseId -> linkToCase(defendantUser, anonymousCaseWorker, letterHolderId, caseId)));
     }
 
+    public void linkDefendantToCaseByCaseId(String caseId, String defendantId) {
+        User anonymousCaseWorker = userService.authenticateAnonymousCaseWorker();
+        this.linkToCaseWithoutRevoking(defendantId, anonymousCaseWorker, caseId);
+    }
+
+    private void linkToCaseWithoutRevoking(String defendantId, User anonymousCaseWorker, String caseId) {
+        LOGGER.info("Granting access to case: {} for user: {}", caseId, defendantId);
```

Comment:
Do all these events need to be at info?
debug level I think would be more appropriate

Reply:
Done